web_src/js/features/stopwatch.js

```diff
@@ -26,28 +25,47 @@ export function initStopwatch() {
     $(this).parent().trigger('submit');
   });
 
-  if (notificationSettings.EventSourceUpdateTime > 0 && !!window.EventSource && window.SharedWorker) {
+  // global stop watch (in the head_navbar), it should always work in any case either the EventSource or the PeriodicPoller is used.
+  const currSeconds = $('.stopwatch-time').attr('data-seconds');
+  if (currSeconds) {
+    updateStopwatchTime(currSeconds);
+  }
+
+  let usingPeriodicPoller = false;
+  const startPeriodicPoller = (timeout) => {
+    if (timeout <= 0 || !Number.isFinite(timeout)) return;
+    usingPeriodicPoller = true;
+    setTimeout(async () => {
+      await updateStopwatchWithCallback(startPeriodicPoller, timeout);
+    }, timeout);
```

Comment:
```suggestion
    setTimeout(() => updateStopwatchWithCallback(startPeriodicPoller, timeout), timeout);
```

Seems useless to `await` if you do nothing after it.